packages/google_maps_flutter/google_maps_flutter/example/ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we can remove all the files in **/xcshareddata/

Reply:
I will revert the `example`. I think `flutter create` was run inside the `example` dir, and it added a lot of stuff that is not needed for this PR.